library/src/main/java/com/github/paolorotolo/appintro/util/TypefaceWorker.java

```diff
@@ -25,26 +28,39 @@ public TypefaceWorker() {
     /**
      * Direct constructor by giving the typeface
      *
-     * @param typeface <p>must be an <b>Integer</b> or a <b>String</b>, first one a FontResource;
-     *                 second one Font Name</p>
-     * @throws RuntimeException <p>when the provided typeface is not an <b>Integer</b> or a <b>String</b></p>
+     * @param typeface <p>a <b>String</b>, which is an URL to font found at Assets folder
      */
-    public TypefaceWorker(Object typeface) {
+    public TypefaceWorker(String typeface) {
```

Comment:
As the constructors have changed, it is not possible to merge pull request without breaking build - working on it...

Reply:
Solved on commit 5ae2863